charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -401,6 +401,10 @@ node:
     runAsUser: 0
     runAsGroup: 0
     fsGroup: 0
+
+  # allows you to deploy aws-ebs-csi-node daemonset to separate namespace (make sure namespace exists before deloy) 
+  namespaceOverride: ""
```

Comment:
Suggest simplifying this parameter to `namespace`. The `Override` postfix does not add helpful context.

Reply:
I did named it `namespaceOverride` because it overrides namespace that you may set with `--namespace` helm flag. Also I saw it named like that in bunch of other charts so it matches common style but, if you insist we can change it to `namespace`

Reply:
That makes sense, `namespaceOverride` works

Reply:
Good, marking this conversation as resolved)